package.json

```diff
@@ -76,7 +76,7 @@
 		"socket.io": "2.0.3",
 		"socketcluster": "14.3.3",
 		"socketcluster-client": "14.2.1",
-		"sodium-native": "LiskHQ/sodium-native#dd0319f",
```

Comment:
It's outside of the scope of this issue for now. We will make the decision next week, whether to keep using the fork or switch to the public `sodium-native` library.